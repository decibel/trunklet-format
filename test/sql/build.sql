\set ECHO none
\i test/pgxntool/psql.sql

BEGIN;
\i test/deps.sql

-- This is just an easy way to get the dependencies pulled in on 10.0+
SELECT pg_temp._create();
DROP EXTENSION "trunklet-format";

\i sql/trunklet-format.sql

\echo # TRANSACTION INTENTIONALLY LEFT OPEN!

-- vi: expandtab sw=2 ts=2
