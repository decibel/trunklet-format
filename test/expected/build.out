\set ECHO none
 template_language__add 
------------------------
 
(1 row)

# TRANSACTION INTENTIONALLY LEFT OPEN!
